src/client/java/teammates/client/scripts/util/LoopHelper.java

```diff
@@ -0,0 +1,39 @@
+package teammates.client.scripts.util;
+
+/**
+ * Utility class to be used with loops to print messages regularly.
+ */
+public class LoopHelper {
```

Comment:
Alternatively, bring back the getCount() method to this class and define the scope as `helping to manage loops` (not just printing).